qemu/target-i386/cpu.h

```diff
@@ -699,7 +699,7 @@ typedef enum {
 
 typedef struct SegmentCache {
     uint32_t selector;
-    target_ulong base;
+    uint64_t base;   /* handle 32 or 64 bit CPUs */
```

Comment:
yes, some parts of code under qemu/ need access to Unicorn struct. but in general, we try to keep the change to Qemu code minimal, and only do if we cannot avoid that.